zerver/lib/exceptions.py

```diff
@@ -375,3 +375,14 @@ def __init__(
         self.errors: List[Tuple[str, str, bool]] = errors
         self.sent_invitations: bool = sent_invitations
         self.license_limit_reached: bool = license_limit_reached
+
+
+class AccessDeniedError(JsonableError):
+    http_status_code = 403
```

Comment:
I think it's probably worth adding a generic code for ACCESS_DENIED errors. It seems possible we should use the official FORBIDDEN term in the description of HTTP 403 instead of "Access denied"?  Maybe start a quick "api design" thread on this question.

Reply:
Posted https://chat.zulip.org/#narrow/stream/378-api-design/topic/ACCESS_DENIED.20or.20FORBIDDEN/near/1225815